my_vars.tfvars

```diff
@@ -0,0 +1,38 @@
+# Dev variables
+project_id          = "focal-freedom-236620"
+environment         = "tmp-alex"
+gcp_region          = "us-west2"
+gcp_service_account = "iofog-platform@focal-freedom-236620.iam.gserviceaccount.com"
+
+# iofog vars
+controller_ip       = "" # Static ip for loadbalancer, eompty is fine.
+# iofog images
+controller_image    = "iofog/controller:1.1.1"
```

Comment:
Do we want users to be able to specify individual versions of everything? My thought is that they could pick 1.0, or 1.1 for iofog stack in general and we manage which images that actually uses in the background